pkgs/tools/networking/prettyping/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "${program}-${version}";
+  program = "prettyping";
+  version = "1.0.1";
+  src = fetchFromGitHub {
+    owner = "denilsonsa";
+    repo = program;
+    rev = "e8d7538b8742b27cffe28e9dfe13d1d1a12288e3";
```

Comment:
Since it's a tagged release, we prefer to use the tag here: `rev="v${version}";` , this makes updating easier